proto/backend/servers.proto

```diff
@@ -0,0 +1,61 @@
+syntax = "proto3";
+
+package rivet.backend.servers;
+
+import "google/protobuf/struct.proto";
+import "proto/common.proto";
+
+message Server {
+    rivet.common.Uuid cluster_id = 1;
+    rivet.common.Uuid datacenter_id = 2;
+    int32 kill_timeout = 3;
+    map<string, MetadataJson> metadata = 4;
```

Comment:
change to just string, string

move to just string, change to validation-level item

Reply:
make json here?

Reply:
just use string